src/doc/book/associated-types.md

```diff
@@ -67,7 +67,7 @@ trait Graph {
 Simple enough. Associated types use the `type` keyword, and go inside the body
 of the trait, with the functions.
 
-These `type` declarations can have all the same thing as functions do. For example,
+These `type` declarations work in the same way as those for functions do. For example,
 if we wanted our `N` type to implement `Display`, so we can print the nodes out,
```

Comment:
I think @tshepang's version is clearer (I assumed the original was just written late at night / in a state of exhaustion)